modules/upgrade-scanner.adoc

```diff
@@ -20,45 +36,98 @@ $ roxctl -e "$ROX_CENTRAL_ADDRESS" scanner generate
+$ oc -n stackrox set image deploy/scanner scanner=registry.redhat.io/advanced-cluster-security/rhacs-scanner-rhel8:{rhacs-version} 
 ----
-<1> If you use Kubernetes, enter `kubectl` instead of `oc`.
+
 . Update the Scanner database image:
 +
 [source,terminal,subs=attributes+]
 ----
-$ oc -n stackrox set image deploy/scanner-db db=registry.redhat.io/advanced-cluster-security/rhacs-scanner-db-rhel8:{rhacs-version} <1>
+$ oc -n stackrox set image deploy/scanner-db db=registry.redhat.io/advanced-cluster-security/rhacs-scanner-db-rhel8:{rhacs-version}
+----
+
+If you use Kubernetes:
+
+. Update the Scanner image:
++
+[source,terminal,subs=attributes+]
+----
+$ kubectl -n stackrox set image deploy/scanner scanner=registry.redhat.io/advanced-cluster-security/rhacs-scanner-rhel8:{rhacs-version} 
 ----
-<1> If you use Kubernetes, enter `kubectl` instead of `oc`.
+
+. Update the Scanner database image:
 +
 [source,terminal,subs=attributes+]
 ----
-$ oc -n stackrox set image deploy/scanner-db init-db=registry.redhat.io/advanced-cluster-security/rhacs-scanner-db-rhel8:{rhacs-version} <1>
+$ kubectl -n stackrox set image deploy/scanner-db db=registry.redhat.io/advanced-cluster-security/rhacs-scanner-db-rhel8:{rhacs-version}
 ----
-<1> If you use Kubernetes, enter `kubectl` instead of `oc`.
 
 .Verification
 
-* Check that the new pods have deployed successfully:
+Check that the new pods have deployed successfully:
+
+* If you use {ocp}:
 +
 [source,terminal]
 ----
-$ oc get pod -n stackrox --watch <1>
+$ oc get pod -n stackrox --watch 
+----
+
+* If you use Kubernetes:
++
+[source,terminal]
+----
+$ kubectl get pod -n stackrox --watch 
+----
+
+
+[id="ugrading-to-371"]
+== Upgrading to RHACS version 3.71
+
+If you are upgrading to RHACS 3.71 using the `roxctl` CLI and YAML.yaml files, you need to perform some additional steps. Scanner DB no longer mounts the `scanner-db-password` Kubernetes Secret into the `db` Scanner DB container. Instead, it is solely used in the `init-container`, `init-db`. Therefore, you must add the environment variable `POSTGRES_PASSWORD_FILE` to the `init-container` configuration. The `init-container` must also mount the `scanner-db-tls-volume` and `scanner-db-password` volumes. The following section provides the upgrade steps for {product-title-short} if you are using {ocp} or Kubernetes.
+
+.Prerequisites
+
+This procedure assumes the `db` container in the Scanner DB configuration is at `index 0` (the first entry in the `containers` list) and the `scanner-db-password` volume mount is at `index 2` (the third entry). This scenario applies to most customers, but you should check the configuration for Scanner DB before entering these commands. If your values differ, you must adjust the `…/volumeMounts/x` value in the following commands. 
+
+.Procedure
+
+* If you use {ocp}, enter the following commands:
++
+[source,terminal]
```

Comment:
Per our slack discussion, I was unable to get the command to cut and paste properly or display properly with `\` and spaces, so I have left it as is for ease of people pasting it directly into their terminal.

Reply:
"If the "common" case is that I only have to copy the command and run it, it might be better to leave the really long command instead of trying to break it up."